lib/util.js

```diff
@@ -371,6 +372,23 @@ function _exceptionWithHostPort(...args) {
   return new ExceptionWithHostPort(...args);
 }
 
+let dotenv;
+
+/**
+ * Loads the .env file from the given path.
+ * @param {string | undefined} path
+ * @returns {void}
+ */
+function loadEnvFile(path) {
```

Comment:
Is it better to call `loadEnvFile(path)` several times for different files, or use `loadEnvFiles([path1, path2])` once for multiple files?

Reply:
This is a use-case that can be added later, if it is common enough.